locales/de.json

```diff
@@ -77,42 +77,42 @@
+	"You can see how it works on ": "Wie das funktioniert, kannst du dir hier ansehen ",
+	"You can choose to enable or disable this option.": "Du kannst wählen, ob du diese Option aktivieren oder daektivieren möchtest.",
 	"Enable analytics to help improve Boostnote": "Datenerhebung zur Verbesserung von Boostnote aktivieren",
 	"Crowdfunding": "Crowdfunding",
-	"Dear everyone,": "Dear everyone,",
-	"Thank you for using Boostnote!": "Thank you for using Boostnote!",
-	"Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote is used in about 200 different countries and regions by an awesome community of developers.",
-	"To continue supporting this growth, and to satisfy community expectations,": "To continue supporting this growth, and to satisfy community expectations,",
-	"we would like to invest more time and resources in this project.": "we would like to invest more time and resources in this project.",
-	"If you like this project and see its potential, you can help by supporting us on OpenCollective!": "If you like this project and see its potential, you can help by supporting us on OpenCollective!",
-	"Thanks,": "Thanks,",
-	"Boostnote maintainers": "Boostnote Betreuer",
-	"Support via OpenCollective": "Support per OpenCollective",
+	"Dear everyone,": "Hallo,",
+	"Thank you for using Boostnote!": "Danke, dass du Boostnote verwendest.",
+	"Boostnote is used in about 200 different countries and regions by an awesome community of developers.": "Boostnote wird in über 200 verschiedenen Ländern von einer großartigen Community von Entwicklern verwendet.",
+	"To continue supporting this growth, and to satisfy community expectations,": "Um die Erwartungen der Community weiterhin erfüllen zu können und die Verbreitung von Boostnote weiter voranzutreiben,",
+	"we would like to invest more time and resources in this project.": "würden wir gern mehr Zeit und Resourcen in dieses Projekt investieren.",
+	"If you like this project and see its potential, you can help by supporting us on OpenCollective!": "Wenn dir dieses Projekt gefällt und du sein Potential erkennst, kannst du uns gern mit OpenCollective unterstützen!",
+	"Thanks,": "Vielen Dank,",
+	"Boostnote maintainers": "Dein Boostnote Team",
+	"Support via OpenCollective": "Unterstützen mit OpenCollective",
 	"Language": "Sprache",
 	"English": "Englisch",
 	"German": "Deutsch",
 	"French": "Französisch",
-	"Show \"Saved to Clipboard\" notification when copying": "\"Auf Clipboard gespeichert\" Benachrichtigungen beim kopieren anzeigen",
+	"Show \"Saved to Clipboard\" notification when copying": "\"Auf Clipboard gespeichert\" Benachrichtigungen beim Kopieren anzeigen",
 	"All Notes": "Alle Notizen",
-	"Starred": "Starred",
+	"Starred": "Markiert",
```

Comment:
How about `Favorisiert`?

Reply:
This is a tricky one. We can discuss this. I looked how it is translated in well-known apps like gmail ("Markiert") or whatsapp ("Mit Stern markierte"). Maybe there are apps out there, where it is translated as "Favoriten"

Reply:
Oh I see. In that case I'm okay with that. :)
Nice research.

Reply:
"Favoriten" sounds good. :D

Reply:
**Starred** is an adjective, so the German translation should be one as well. **Favoriten** is a noun.

Furthermore you could simply want to _mark_ without it being your _favorite_.
I'm okay with **Markiert** (Marked).